### ISSUE TITLE
Haven't had this happen in a while, but my Kodi is frozen on loading screen. any help please?

### ISSUE THREAD

**Issue by frankee123** [dobbelina/repository.dobbelina#819]
I'm sorry as this isn't a cumination issue, but you guys know your stuff here so I figured I'd ask here if it's ok to do so. This has happened numerous times to me even on my older android boxes where kodi would just freeze on the boot up screen. There was nothing I could do to fix it. I would force close kodi, clear cache, etc. It would still freeze when I tried to open it up again. It always led me to uninstalling kodi and re-installing it to work again. I just hate doing this since I have to put my custom sites and things to the main menu of cumination. Does anybody know of a working fix before I just uninstall and reinstall kodi please? I also will add I googled the problem and seen people saying  Rename or delete the guisettings.xml file. Only issue is when I go into my android settings into my kodi folder, everything is blank. There is nothing at all which seems odd to me. Maybe if somebody knows of a way to somehow fix this as well it could help me. Thank you so much.

**Comment by dobbelina** [comment]
If the kodi folder is blank something must have been corrupted.
Everytime you start kodi it creates a logfile in the kodi folder, don't you have that to?

**Comment by frankee123** [comment]
Ty for the response sir. It's very odd it shows nothing in my kodi folder when I go into it. I will have to show a pic when I get a chance soon. It's not even my kodi file, I don't see anything in any of my files on my android box.

**Comment by frankee123** [comment]
![Uploading 20220728_225346.jpg…]()

**Comment by dobbelina** [comment]
Link above broken.
Just drag and drop the picture into the text edit field and wait for it to upload, then press "Comment"

**Comment by frankee123** [comment]
![20220728_225346](https://user-images.githubusercontent.com/60049371/181786219-015fe1a3-e486-4614-b2d6-32043e1f0602.jpg)

**Comment by frankee123** [comment]
Sorry about that. I was reading about my issue and it may actually be something they changed in android 11 that they won't let you look at any files at all or something. I really don't know what to do. It just really sucks to uninstall cumination and reinstall and just add all of the main menu items I had really. It's not so bad to reimport my favorites and keywords. I'll wait for your next response, but I will most likely end up reinstalling cumination later on I guess. Ty

**Comment by dobbelina** [comment]
Kodi folder is hidden in Android, read this https://forum.kodi.tv/showthread.php?tid=320443
and https://ccm.net/mobile/android/923-how-to-display-hidden-files-on-android/

**Comment by frankee123** [comment]
Thanks a lot bud, but i dont have a file manager and menu settings like it looks like an android phone on the second site you listed. I just dont see anything on there that helps me fix my issue. Maybe this android box I got is faulty or something. This really sucks to uninstall kodi and do reinstall it since I think this will eventually happen again sadly. I really appreciate your time and effort though. It's much appreciated.

**Comment by dobbelina** [comment]
The picture you posted is from a file manager.
Have you checked it's settings if you can choose to see hidden files?
Otherwise just download one that can.

**Comment by frankee123** [comment]
Yea bud, there is no settings on my android box or on my remote is what I was trying to get at. I did try to download files by google but there was nothing new to look at as well so I just gave up. I think it could be something new with android 11 is what I figured out reading about it. Definitely weird, annoying, and frustrating. I ended up clearing my kodi data and started again from fresh. Do you have any idea as to why my kodi would shut down when I try to click on my download folder though? also, is it possible to have a setting that will save my main menu items like I can with my favorites and keywords too? Thanks man, you can close this if you respond to it again.

**Comment by DJMix-I-Can** [comment]
Android 11 definitely restricted it. I installed the Nvidia Experience 9.0 (Android 11 update) on my shield & it screwed my back up process with FX Explorer. I did some reading & a lot of people mentioned that using CX File Explorer was able to view the .kodi folder. I installed it and was able to view the kodi folder. Regardless of it working there were other issues after the Nvidia update. I wound up rolling back to the previous Nvidia Experience. (Android 9)  As for backing up, I usually go to userdata/thumbails & delete all of the folders. That's where are the pics are stored. If you haven't done that in a long time, that will take up space. Then I'll go to addons/packages and move them to my USB for back up. If this folder doesn't get cleaned out after a long time and have had numerous updates, those zip files will take up some space. Once I've done all of that, I go back to first folder which .kodi. I long press on it and archive/zip it up. Once it's done, I move that zip files to my USB as a backup. If my Kodi installation ever comes corrupt for some reason, I'll extract that backup & won't have to reinstall anything. As for the download folder, it might an Android 11 permission issue.

**Comment by frankee123** [comment]
Thx a lot for the response DJ. I tried installing one called files by google but I wasnt able to see anything so i uninstalled it. I will install the cx file explorer as you suggest. I did also install a maintenance app on kodi before I had the freezing issue that did delete the images. I'm really not sure what caused the freezing issue to happen. All I can say is whenever I try to view my movies in my cumination download folder, kodi would close itself out. It would only work if I reopened kodi right away and clicked again to view my download folder. This is what happened though when I tried to reopen kodi after this happened, and it gave me the constant freeze on the loading screen. I don't mean to bother anyone. I don't know much about how to fix things and don't want to be a bother here, I just couldn't find an a reason why it was happening. Thank you again for the help!

**Comment by DJMix-I-Can** [comment]
Did you always have Android 11 on the Q9 4K box, or did you get a firmware update for it between when it was working & then it didn't.

**Comment by frankee123** [comment]
No, it was always on it. It is pretty new still. I got it off amazon. It gave me a previous issue which I needed help on here to fix that was another issue with it having android 11.

**Comment by frankee123** [comment]
So I was just testing out the cx file explorer but it keeps giving me this error everytime i try to go into my android folder. I can't believe how annoying it is to look at all my files. It pops up saying please allow the app to access storage first by following instructions below. It says tap "Use this folder" button on the next screen. I hit ok but it says loading error in the background really quick and also application not found on the bottom screen really quick.  if i x out the box it says loading error! Access denied. what is also weird is if i go under my settings to look at all of my apps, click on cx file explorer to click on permissons at the bottom, nothing happens and it just kicks me to the main menu on my android box. I'm guessing that is what needs to be done. Maybe for some reason, the app doesn't have permission to look at all my files, but I do not see a way to fix it. UUUGGHHH. I will just have to bear with clearing the entire kodi caches if it freezes on me again i guess. Ty.

**Comment by DJMix-I-Can** [comment]
Disclaimer: I'm not suggesting to do this. 
If I was in your situation, this is what I would do.
First, If I have any SD card or USB attached, I'd remove them. Then I'd restart the device & see if it's still performing the same way. If it doesn't, then I'd install the SD card or USB again. One at a time, not at the same time. I'll try to determine if one of these peripheral devices is causing a lag on my device. They could be failing. If I don't see any improvement, then I'll see if there's a firmware update for that device. If there isn't, then I'd start backing up anything I have locally from my device. The reason being is because I'll do a factory reset on that device. Once I do the factory reset, I'll install a Vanilla Kodi installation.  I won't install a skin and install one add on. I'll link up my debrid account with that addon and run tests to see how it performs. If everything is working the way it's supposed to work, then I'll add another addon & so on & so on. Once I feel that all my addons are performing the way their supposed to work, then I'll add a skin. I believe you get the idea on what I'm doing, I'm trying to isolate the problem. It's a tedious task but it has to be done in order to isolate the problem. I would've also installed Stremio or Syncler to determine if it's a Kodi issue or device issue. Like I stated before, it's a tedious task but hopefully it would help isolate the problem.....

**Comment by frankee123** [comment]
Thanks bud. I actually have had this same issue with kodi freezing on my previous android box where I just ended up uninstalling it and reinstalling it. This is like a brand new android 11 box (2 months old roughly) where I only have kodi installed on it. I do not have a sd or usb attached. I just have the worst luck with all electronics in all my damn life. Something always goes wrong whether it is an android box, tv, computer, xbox, etc. I already had to clear data on my kodi and start fresh and put cumination back on since I couldn't fix getting past the frozen kodi screen. I know eventually this issue will probably happen out of the blue again to me, but I'm actually going to try not to click on the cumination download folder anymore to see if hopefully it doesn't happen again.
